python/BuiltInExample1.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
+
+from paraview.simple import *
+
+# create a new 'XML Image Data Reader'
+builtInExamplevti = XMLImageDataReader(FileName=['BuiltInExample1.vti'])
+builtInExamplevti.PointArrayStatus = ['Vectors_']
+
+# create a new 'Transform'
+transform1 = Transform(Input=builtInExamplevti)
+transform1.Transform = 'Transform'
+
+# init the 'Transform' selected for 'Transform'
+transform1.Transform.Rotate = [0.0, 0.0, -90.0]
+
+# create a new 'Compute Derivatives'
+computeDerivatives1 = ComputeDerivatives(Input=transform1)
+computeDerivatives1.Scalars = [None, '']
+computeDerivatives1.Vectors = ['POINTS', 'Vectors_']
+computeDerivatives1.OutputVectorType = 'Vorticity'
+
+# create a new 'Cell Data to Point Data'
+cellDatatoPointData1 = CellDatatoPointData(Input=computeDerivatives1)
+
+# create a new 'Calculator'
+calculator1 = Calculator(Input=cellDatatoPointData1)
+calculator1.ResultArrayName = 'myVorticity'
+calculator1.Function = 'Vorticity_Z'
+
+# create a new 'TTK ScalarFieldNormalizer'
+tTKScalarFieldNormalizer1 = TTKScalarFieldNormalizer(Input=calculator1)
+tTKScalarFieldNormalizer1.ScalarField = ['POINTS', 'myVorticity']
+
+# create a new 'Tetrahedralize'
+tetrahedralize1 = Tetrahedralize(Input=tTKScalarFieldNormalizer1)
+
+# create a new 'TTK PersistenceDiagram'
+tTKPersistenceDiagram1 = TTKPersistenceDiagram(Input=tetrahedralize1)
+tTKPersistenceDiagram1.ScalarField = ['POINTS', 'myVorticity']
+tTKPersistenceDiagram1.InputOffsetField = [None, '']
```

Comment:
I don't believe this line is necessary.
Thanks for double checking (e.g. with diff or paraview) that the outputs of the script are unchanged after the edit.